scripts/sct_download_data.py

```diff
@@ -110,7 +111,8 @@ def main(args=None):
         'deepseg_sc_models': ['https://osf.io/avf97/?action=download',
                               'https://www.neuro.polymtl.ca/_media/downloads/sct/20180610_deepseg_sc_models.zip'],
         'deepseg_lesion_models': ['https://osf.io/eg7v9/?action=download',
-                              'https://www.neuro.polymtl.ca/_media/downloads/sct/20180613_deepseg_lesion_models.zip']
+                              'https://www.neuro.polymtl.ca/_media/downloads/sct/20180613_deepseg_lesion_models.zip'],
+        'c2c3_disc_models': ['https://www.neuro.polymtl.ca/_media/downloads/sct/20181129_c2c3_disc_models.zip']
```

Comment:
it would be important to add a README in the zip that lists the subjects used to generate the model (for replicability)